platforms.json

```diff
@@ -277,6 +277,10 @@
     "22.04": {
       "deprecated": false,
       "path": "lib/fauxhai/platforms/ubuntu/22.04.json"
+    },
+    "24.04": {
+      "deprecated": false,
+      "path": "lib/fauxhai/platforms/ubuntu/22.04.json"
```

Comment:
@nickpegg  - Is this supposed to be `"path": "lib/fauxhai/platforms/ubuntu/24.04.json"` ??

Reply:
Good catch! That was a copy-and-paste error on my part